core/platform/models.py

```diff
@@ -27,8 +27,17 @@
 
 
 class _Platform(object):
+    """A base class for platform-specific imports related to GAE.
+    """
```

Comment:
Move this to the end of the previous line. Also, like in other files, add a blank line between the top docstring and the `@classmethod`.